<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>893</width>
    <height>893</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StatsV2</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:rgb(240, 240, 240);

}
QHeaderView::section{
background:transparent; 

}

QPushButton{
background: rgb(226, 226, 226)

}

QComboBox{
background: rgb(226, 226, 226)

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="progTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>871</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Две выборки</string>
     </attribute>
     <widget class="QTableWidget" name="statistic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>400</width>
        <height>569</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>18</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>199</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>199</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Первая выборка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Вторая выборка</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QListWidget" name="onelist">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>320</y>
        <width>200</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QListWidget" name="twolist">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>320</y>
        <width>200</width>
        <height>301</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QListWidget" name="alllist">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QPushButton" name="loadButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузка</string>
      </property>
     </widget>
     <widget class="QComboBox" name="svaz">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Несвязанные выборки</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Связанные выборки</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="resizeButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Размер</string>
      </property>
     </widget>
     <widget class="QPushButton" name="execButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчитать</string>
      </property>
     </widget>
     <widget class="QLabel" name="vi1">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>200</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Первая выборка</string>
      </property>
     </widget>
     <widget class="QLabel" name="vi2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>300</y>
        <width>200</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Вторая выборка</string>
      </property>
     </widget>
     <widget class="QPushButton" name="execButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>График</string>
      </property>
     </widget>
     <zorder>statistic</zorder>
     <zorder>twolist</zorder>
     <zorder>alllist</zorder>
     <zorder>loadButton</zorder>
     <zorder>svaz</zorder>
     <zorder>resizeButton</zorder>
     <zorder>execButton</zorder>
     <zorder>onelist</zorder>
     <zorder>vi1</zorder>
     <zorder>vi2</zorder>
     <zorder>execButton_2</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QCheckBox" name="styleCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Новый интерфейс</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Информация</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>841</width>
        <height>691</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Создатель: Евгений Петров (Render, Red Ender)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;vk.com/red_end&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;jekap505@yandex.ru&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
